### PR TITLE
Added Feature to start/stop/pause zpool scrubbing

### PR DESCRIPTION
**Pull request by Iyer-VivekAnand**
Added a method to start/stop/pause zpool scrubbing.

### CODE REVIEW

File: zpool.go [hunk 1]
```diff
@@ -1055,6 +1064,41 @@ func (pool *Pool) VDevTree() (vdevs VDevTree, err error) {
...
+	}
+
+	var scanType C.pool_scan_func_t = C.POOL_SCAN_SCRUB
+	var scrubCmd C.pool_scrub_cmd_t = C.POOL_SCRUB_NORMAL
+
+	switch cmd {
+	case PoolScrubStop:
+		scanType = C.POOL_SCAN_NONE
+	case PoolScrubStart:
+		break
```
Comment (sivachandran) [inline zpool.go]: by default Go's switch doesn't fall through. what are we trying to do here?

**Comment by ananthb** [comment]
@Iyer-VivekAnand if a scrub is already running (maybe it was created by this method), can it be stopped by this method as well?

Is this valid usage of this code?
```
pool.Start(PoolScrubStart)
...
pool.Start(PoolScrubStop)
```

**Review by ananthb — COMMENTED** [review]
@Iyer-VivekAnand if a scrub is already running (maybe it was created by this method), can it be stopped by this method as well?

Is this valid usage of this code?
```
pool.Start(PoolScrubStart)
...
pool.Start(PoolScrubStop)
```

File: zpool.go (hunk 1 above)
Comment (Iyer-VivekAnand) [reply zpool.go]: Yes, we dont need the break. I dont want the switch to fall to default for PoolScrubStart.

**Comment by Iyer-VivekAnand** [comment]
> @Iyer-VivekAnand if a scrub is already running (maybe it was created by this method), can it be stopped by this method as well?
> 
> Is this valid usage of this code?
> 
> ```
> pool.Start(PoolScrubStart)
> ...
> pool.Start(PoolScrubStop)
> ```
Yes . Already running scrub can be stopped.
Thats why I have the `PoolScrubStop` command

File: zpool.go [hunk 2]
```diff
@@ -1081,10 +1081,9 @@ func (pool *Pool) Scrub(cmd PoolScrubCmd) (err error) {
 	var scrubCmd C.pool_scrub_cmd_t = C.POOL_SCRUB_NORMAL
 
 	switch cmd {
+        case PoolScrubStart:
```
Comment (sivachandran) [inline zpool.go]: fix indentation.

**Review by sivachandran — APPROVED** [review]

**Review by ananthb — APPROVED** [review]

**Pull request merged**